M/mdbtools/build_tarballs.jl

```diff
@@ -0,0 +1,66 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "mdbtools"
+version = v"0.8.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/cyberemissary/mdbtools.git", "b753ff36a0f1d88ae8a300ed6712f4aa2ddb7d08")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd mdbtools/
+autoreconf -if
+apk add glib-dev
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-man
+make -j8
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:i686, libc=:glibc),
+    Linux(:x86_64, libc=:glibc),
+    Linux(:aarch64, libc=:glibc),
+    Linux(:armv7l, libc=:glibc, call_abi=:eabihf),
+    Linux(:powerpc64le, libc=:glibc),
+    Linux(:i686, libc=:musl),
+    Linux(:x86_64, libc=:musl),
+    Linux(:aarch64, libc=:musl),
+    Linux(:armv7l, libc=:musl, call_abi=:eabihf)
+    MacOS(:x86_64)
+    FreeBSD(:x86_64)
+    Windows(:i686)
+    Windows(:x86_64)
```

Comment:
```suggestion
    Linux(:armv7l, libc=:musl, call_abi=:eabihf),
    MacOS(:x86_64),
    FreeBSD(:x86_64),
    Windows(:i686),
    Windows(:x86_64),
```

Reply:
Aren't these all supported platforms?

Reply:
Yes, you are right. I was only able to test Linux on my local machine, because MacOS and Windows still don't work through BinaryBuilder. If the CI passes, I'll change it to `supported_platforms()`.

Reply:
Windows doesn't seem to work.

Reply:
I'm not going to spend time in investigating this, especially not this week.  It's up to you to decide whether to try and fix it or drop support for Windows.

Reply:
I've dropped support for Windows. For windows there's already prebuilt binaries and I needed this just for Mac and Linux at the moment.

Reply:
I'll open a new PR if I want to add Windows support in the future. Resolved in https://github.com/JuliaPackaging/Yggdrasil/pull/1322/commits/5d7f1abddc90fba117892a0fbe907ee82af32036.